asab/log.py

```diff
@@ -428,10 +427,29 @@ def __init__(self, fmt=None, datefmt=None, style='%', sd_id='sd'):
 		self.converter = time.gmtime
 
 
-class JSONFormatter(StructuredDataFormatter):
+class JSONFormatter(logging.Formatter):
+
+	INSTANCE_ID = os.environ.get("INSTANCE_ID")
```

Comment:
this is crucial for analysis of the data. it should be added to the log in other formatters, too. OR why not to have it in a logger (i.e. BEFORE it gets to formatting)

Reply:
nope, we dont want to have the ids in console